src/resources/MachineLearning/DNEConfiguration/DNEConfiguration.ts

```diff
@@ -32,7 +32,7 @@ export default class DNEConfiguration extends Resource {
 
     parseDocumentExtractionQuery(query: string) {
         return this.api.get<DocumentExtractionQueryModel>(
-            this.buildPath(`${DNEConfiguration.baseUrl}/documentextractionquery`, {query})
+            this.buildPath(`${DNEConfiguration.baseUrl}/documentextractionquerymodel`, {query: decodeURI(query)})
```

Comment:
Is it still relevant to decode?

Reply:
Personally I don't think so. Here you're sending a decoded query to the backend which mean the developer using the method would have to encode it in his code for the decode to have an effect. Since nothing in the method name or parameter tell him to encode it probably won't happen
```suggestion
            this.buildPath(`${DNEConfiguration.baseUrl}/documentextractionquerymodel`, {query})
```